packages/safe-tools-client/app/components/schedule-payment-form-action-card/index.css

```diff
@@ -6,6 +6,10 @@
   --boxel-field-label-align: center;
 }
 
+.schedule-payment-form-action-card .boxel-field.schedule-payment-form-action-card__frequency {
+  --boxel-field-label-align: top;
+}
```

Comment:
The design has the label centre-aligned:

<img width="621" alt="Schedule Payment - Fill in Info - One Time Payment - MVP - Zeplin 2022-11-10 10-44-39" src="https://user-images.githubusercontent.com/43280/201155553-0166e2b7-3308-48c9-98bc-2dbe2cd7d845.png">

<img width="722" alt="Cardstack - Percy | Visual testing as a service 2022-11-10 10-43-53" src="https://user-images.githubusercontent.com/43280/201155482-d3b47d36-217e-4305-bff6-ed2f742d6858.png">